app/controllers/posts_controller.rb

```diff
@@ -16,5 +16,33 @@ def index
 
   def show
     @post = Post.find(params[:id])
+    @current_user = current_user
+    @like = Like.new
```

Comment:
Hi @HAMMAS-SALEEM, 

Thanks for highlighting the issue but I do not exactly understand why it's throwing an error because it's working fine on my side.

Reply:
OK Dear if it is working well on your side then the next reviewer can verify it 👍